hack/addons/kustomize/nutanix-storage-csi/helm-values.yaml

```diff
@@ -1,4 +1,4 @@
 # Copyright 2023 D2iQ, Inc. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-createSecret: false
```

Comment:
I don't think this file is needed since you removed charts/cluster-api-runtime-extensions-nutanix/templates/csi/nutanix/manifests/nutanix-storage-csi-configmap.yaml?

Reply:
ah actually its not. good catch.